tests/unit/mocks.py

```diff
@@ -62,7 +64,11 @@ def expired(self) -> bool:
         """
         if self.expiry is None:
             return False
-        return False if self.expiry > datetime.datetime.utcnow() else True
+        return (
```

Comment:
Could this be better expressed as:

```
if self.expiry > datetime.datetime.now(datetime.timezone.utc):
  return False

return True
```

?

Reply:
yup, had it all one line but looks like formatter edited it